### PR TITLE
JS DOM API task

### PR DESCRIPTION
**Pull request by rrrds**
### JS DOM API

[Demo](https://rrrds.github.io/kottans-hw-dom/) |
[Code base](https://github.com/rrrds/kottans-hw-dom)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/rrrds/js-dom/js/main.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  content.dataset.indexNumber = idx;
+
+  main.appendChild(content);
+});
+
+navigation.appendChild(ul);
+
+document
+  .querySelector('.nav__item')
+  .dispatchEvent(new MouseEvent('click', { view: window, bubbles: true }));
```
Comment (zonzujiro) [inline submissions/rrrds/js-dom/js/main.js]: What for?

Comment (rrrds) [reply submissions/rrrds/js-dom/js/main.js]: To trigger first menus' item and set initial content view

Comment (zonzujiro) [reply submissions/rrrds/js-dom/js/main.js]: Why not make this with an initial state at HTML file?

Comment (rrrds) [reply submissions/rrrds/js-dom/js/main.js]: Set initial state while adding new content to page

**Pull request merged**